### ISSUE TITLE
Martialart fencing allows Cudgel

### ISSUE THREAD

**Issue by ZenZen2** [CleverRaven/Cataclysm-DDA#23470]
![cudgel](https://user-images.githubusercontent.com/20055252/38715373-119764b2-3edb-11e8-9a5f-060830aa26a3.jpg)
Cudgel is useable with fencing, is this a mistake?

Edit: As @aeoo suggested reducing its volume down to 1.25L and perhaps a rename/description change would be a good solution?

**Comment by terribleperson** [comment]
The description of the cudgel is "A slender long rod of wood, while traditionally intended as a training tool for many dueling moves, it still makes a good melee weapon in a pinch." It's essentially a practice sword, so it makes sense for fencing to allow it.

**Comment by smolbird** [comment]
Basically, think singlestick, not club.

**Comment by Zireael07** [comment]
Maybe rename cudgel to practice sword or something?

**Comment by terribleperson** [comment]
Google 'cudgel fencing'. Cudgel is an appropriate name.

**Comment by aeoo** [comment]
From the wikipedia page for 'singlestick':

>The singlestick itself is a slender, round wooden rod, traditionally of ash, with a basket hilt. Singlesticks are typically around 36 inches (91 cm) in length and 1 inch (2.5 cm) in diameter and thicker at one end than the other. It bears approximately the same relationship to the backsword as the foil to the small sword in being a sporting version of the weapon for safe practice.

Should cudgels be slimmed down in-game?  I think the confusion arises from how massive a cudgel is as it is presently define in-game:

2L, 1.93lbs.

The weight seems OK, but 2 liters is questionable when a cavalry saber is:

1.25L and a bit heavier.

So a cudgel is almost twice the size of a cavalry saber.

A heavy stick is 1.25L

A two by four is 1.50L

A two by sword is 1.25L

In other words, cudgel seems to be way larger than the thing it's made of and also larger than a cavalry saber.  This might be what makes people think it's either a club or a staff or something like that, instead of a practice sword.

Maybe cudgel should be toned down to 1.25L to bring it line with the size of a cavalry sword as well as making it in line with the size of the source material from which it's made, which is one heavy stick or one two by four.

It's actually even an exploit of sorts I've been quietly using if I want more splintered wood, first I convert a heavy stick into a cudgel, and then I cut it, and I magically get more splintered wood that way.

So there are all sorts of reasons to tone down the size of the cudgel to something more realistic.  It will avoid all this confusion and it will also eliminate the exploit of magically allowing more splintered wood to be made from a single stick.

**Comment by smolbird** [comment]
Since cudgels are somewhat underwhelming damage-wise, that might be a decent option.

**Comment by MattD94** [comment]
I've always thought of the cudgel in game as too large and far too slow. An actual cudgel would not likely be as long as a cavalry saber, and certainly not nearly as slow to use it is in game.

When I did a google image search for cudgel, most of the images were of fantasy like weapons that were too large or exaggerated to compare to the real life item. Additionally, although this may just be due to the similarities of terms used by google image, when I searched cudgel fencing, some of the images were those of shortish quarterstaves.

Also if we are considering sheathing cudgels and whatnot, I think it would be nice in the future to allow sheathing hatchets or maces in the same way the hallagin bar is sheathed in the firefighter belt (as well as a wrench in a tool belt maybe). Maybe sheathing something like a hatchet/crowbar/etc on a special belt/harness would incur more encumbrance than a sword in a scabbard, but real life hammers and axes used for battle had very small heads compared to what is commonly depicted in games and fantasy art and should totally hang from a belt or lanyard just fine. Of course this should be a separate issue, or maybe just a "realistic historic arms mod".
